profiles/markaspot/themes/ntxuva/js/ntxuva.js

```diff
@@ -806,7 +808,7 @@ var lang_pt = {
                 return;
 
             $.ajax({
-                url: '/georeport/v2/requests.xml',
+                url: '/mopa/public/georeport/v2/requests.xml?start_date='+defaultStartDate,
```

Comment:
here the path URL was changed not only to start from the new default data but also take the data from a different path. I think all we want is the default date part.